generative_ai/context_caching/get_list_of_content_caches.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+from typing import List
+
+from google.cloud.aiplatform_v1beta1.types.cached_content import CachedContent
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+
+
+def get_list_of_context_caches() -> List[CachedContent]:
+    # [START generativeaionvertexai_gemini_get_list_of_context_caches]
+    import json
+
+    from datetime import datetime as dt, timezone as tz
+
+    import vertexai
+
+    from vertexai.preview import caching
+
+    # TODO(developer): Update project id and location
+    vertexai.init(project=PROJECT_ID, location="us-central1")
+
+    cached_content_list = caching.CachedContent.list()
+    # Access individual properties of a CachedContent object
+    for cc in cached_content_list:
+        expire_time_in_seconds = int((cc.expire_time - dt.now(tz.utc)).total_seconds())
```

Comment:
Instead of doing this calculation, I'd recommend either printing the timestamp directly or use this library https://pypi.org/project/human-readable/

Reply:
Thanks! I don't think it justifies to pull a library for just one nice print, so I will use the value of `.expire_time` field directly.